lua/nvim_lsp/pyright.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+local server_name = "pyright"
+
+local installer = util.npm_installer {
+  server_name = server_name;
+  packages = {server_name};
+  binaries = {server_name};
+}
+
+configs[server_name] = {
+  default_config = {
+    cmd = {"pyright-langserver", "--stdio"};
+    filetypes = {"python"};
+    root_dir = util.root_pattern(".git");
```

Comment:
Maybe include some more common root patterns, e.g. `setup.py`, `setup.cfg`, `pyproject.toml` and `requirements.txt`?

Reply:
Done, thanks for the suggestion!